### PR TITLE
 Pages tagged with place get a full-width map across the top of the page (instead of in the sidebar)

### PR DESCRIPTION
**Pull request by sagarpreet-chadha**
Continued from https://github.com/publiclab/leaflet-blurred-location-display/issues/64#issuecomment-482655146 😄 !

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

### Screenshots : 

*   **With `place` tag :** 

Screenshot 2019-04-18 at 2 53 11 PM

*   **Without `place` tag :** 

Screenshot 2019-04-18 at 3 04 31 PM

*   **human-readable placename is already added as a tag via LBL model :** 

Screenshot 2019-04-18 at 3 07 28 PM

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
This looks FANTASTIC!

This has gotten broken a few times... do you think you could write a simple system test for it? #5316 Then we can protect it against future changes? I think it could be pretty easy to add. Thank you!!!

**Comment by sagarpreet-chadha** [comment]
Okay `assert_select "div"` is not working . 
Meaning `assert_select` does not work in system testing .

**Comment by sagarpreet-chadha** [comment]
Tests passed ❤️ . Ready to be merged .
@jywarren ...kindly review 😄 !!!

**Comment by jywarren** [comment]
PERFECT awesome.

File: test/system/place_tags_test.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require "application_system_test_case"
+
+class PlaceTagsTest < ApplicationSystemTestCase
+
+  # node blog has lat and lon tag and place tag .	
+  test "pages tagged with place get a full-width map across the top of the page" do
```
Comment (jywarren) [inline test/system/place_tags_test.rb]: Love this!!!

Comment (jywarren) [reply test/system/place_tags_test.rb]: Hey, @sagarpreet-chadha can you see if I'm doing anything wrong here? https://github.com/publiclab/plots2/pull/5605/

**Pull request merged**